SpatialGDK/Documentation/content/tutorials/dbsync/tutorial-dbsync-intro.md

```diff
@@ -2,11 +2,11 @@
 
 ## Database Sync Worker Tutorial
 
-This tutorial takes you through how to integrate the [Database Sync Worker Example](https://github.com/spatialos/database_sync_worker) in a GDK project and use it to store persistent data outside of a SpatialOS deployment. 
+This tutorial takes you through how to integrate the [Database Sync Worker Example](https://github.com/spatialos/database-sync-worker) in a GDK project and use it to store persistent data outside of a SpatialOS deployment.
 
-The Database Sync Worker is a SpatialOS server-worker designed to easily sync and persist cross-session game data (such as player inventories) between SpatialOS and an external database.
+The Database Sync Worker is a SpatialOS server-worker designed to easily synchronise and persist cross-session game data (such as player inventories) between SpatialOS and an external database.
```

Comment:
```suggestion
The Database Sync Worker is a SpatialOS server-worker designed to easily synchronize and persist cross-session game data (such as player inventories) between SpatialOS and an external database.
```

-ize is used in docs I think, stop trying to use British spelling Jared, embrace your heritage